### PR TITLE
Add buildbot build commands for the hybrid examples and remove redundant checks

### PR DESCRIPTION
**Pull request by drpdr**
Let's make sure we can at least build the hybrid examples. We have basic assertions for testing but `.buildbot.sh` is missing a build check.

### CODE REVIEW

File: .buildbot.sh [hunk 1]
```diff
@@ -35,6 +35,23 @@ for dir in . employee shared_objects timsort; do
     popd
 done
 
+echo "Checking that all the hybrid examples build on Morello..."
+platform='morello-hybrid'
+for dir in hybrid; do
+    pushd "$dir"
+        make -f Makefile.$platform clean
+        if [ -d bin/$platform ]; then
```
Comment (ltratt) [inline .buildbot.sh:43]: Is this check really necessary? It seems *very* paranoid!

File: .buildbot.sh [hunk 2]
```diff
@@ -35,6 +35,23 @@ for dir in . employee shared_objects timsort; do
...
+            echo "Error: 'make clean' did not clean the output directory."
+            false
+        fi
+        make -f Makefile.$platform all
+        if [ ! -d bin/$platform ]; then
```
Comment (ltratt) [inline .buildbot.sh:48]: Same comment as above.

File: .buildbot.sh (hunk 1 above)
Comment (drpdr) [reply .buildbot.sh:43]: I have copied from the previous PR. I may agree with you, though.

Comment (ltratt) [reply .buildbot.sh:43]: OK, let's kill these checks and then in another commit kill the other checks too?

Comment (drpdr) [reply .buildbot.sh:43]: Agreed.

Comment (drpdr) [reply .buildbot.sh]: Done in 49c7009.

**Comment by ltratt** [comment]
Please squash.

**Comment by drpdr** [comment]
Squashed.

File: .buildbot.sh (hunk 1 above)
Comment (drpdr) [reply .buildbot.sh]: > OK, let's kill these checks and then in another commit kill the other checks too?

Sorry, I have just realised you said in another *commit* not PR. I had to kill the other checks too here, then...

**Comment by drpdr** [comment]
Wait I can rename the PR in "... and fix buildbot redundant checks", so I have a separate commit for the redundant check removal. What do you think?

**Comment by ltratt** [comment]
Push that commit here as a second commit please.

**Comment by drpdr** [comment]
> Push that commit here as a second commit please.

Done.

**Comment by drpdr** [comment]
Ah, I had tested it but, of course, I had my `bin`.

**Comment by drpdr** [comment]
bors try

**Comment by ltratt** [comment]
Please squash.

**Comment by drpdr** [comment]
Squashed.

**Pull request merged**